lighthouse-core/scripts/publish-test.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+TXT_BOLD=$(tput bold)
+TXT_DIM=$(tput setaf 245)
+TXT_RESET=$(tput sgr0)
+
+DIRNAME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+LH_PRISTINE_ROOT="$DIRNAME/../../../lighthouse-pristine"
+
+set -euxo pipefail
+
+bash "$DIRNAME/publish-prepare-pristine.sh"
+
+cd "$LH_PRISTINE_ROOT"
+
+# Install deps
+yarn --check-files
+
+# Test err'thing
+echo "${TXT_BOLD}Building all the clients..."
+yarn build-all
+
+echo "Running the standard test suite..."
+yarn test
+
+echo "Running the smoke tests...."
+yarn smoke
```

Comment:
dunno about @hoten but i usually run this stuff in another shell since it's so time-consuming.

not sure about a good solution for parallelizing, though.
https://stackoverflow.com/a/38109901/89484 perhaps?

Reply:
yeah parallel shell business always makes me nervous... good candidate for a perf OYB testing PR? :D